crates/nu-cmd-lang/src/example_test.rs

```diff
@@ -61,19 +64,27 @@ mod test_examples {
             // Try to keep this working set small to keep tests running as fast as possible
             let mut working_set = StateWorkingSet::new(&engine_state);
             working_set.add_decl(Box::new(Break));
+            working_set.add_decl(Box::new(Collect));
```

Comment:
Are all these new `add_decl`'s needed for this PR? We try to keep these at a minimum and not allow every command to be tested in examples.

Reply:
I suppose part of the reason I opened this PR is that I'm wondering which examples you do and don't want to test. To test all the existing examples of `use`, yes all those `add_decl` are needed.

Reply:
Good question. I'm not sure exactly myself. Let's see what others say.